index.html

```diff
@@ -164,6 +164,22 @@ <h2 class="card-title">
         </div>
     </div>
         </section>
+        <section class="contact-me display-flex flex-column align-middle">
+            <h1 class="section-heading neutrals txt-center">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. 
+            </h1>
+            <div class="separator-blank"></div>
+            <form class="my-contact-form display-flex flex-column align-middle"
+            action="https://formspree.io/f/xdojwqqv"
+            method="POST"
+            >
+                <input type="text" name="name" maxlength="30" placeholder="Full name">
+                <input type="email" name="email" placeholder="Email Adress">
+                <textarea name="message" placeholder="Enter text here" maxlength="500"></textarea>
+                <button class="button btn-primary border-none" type="submit">Get In touch</button>
+            </form>
```

Comment:
- [ ] Please make sure that all input fields have the `required` attribute to activate the validation on submission.
   Since you did not validate your form by adding the `required` attribute to each input field, this is what happens when I tried to submit an empty form.
![Screenshot from 2022-11-01 15-30-25](https://user-images.githubusercontent.com/92303891/199272933-a7a88d4c-8f86-4953-9e72-d4393423e841.png)